### PR TITLE
[#6] Shop page

### PR DESCRIPTION
**Pull request by mjordonez2001**
### Issues
- #6 

### What?
- Maps through all supplements and displays name, price, and placeholder image

### Screenshots
Screen Shot 2022-11-26 at 7 17 45 PM

### CODE REVIEW

File: src/pages/shop/Supplement.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  benefits?: string[];
+};
+
+function Supplement({ name, slug, unit_price, photo_url }: supplementProps) {
+  return (
+    <Link to={`/${slug}`} className="text-decoration-none text-dark">
+      <div className="card">
+        <img src={photo_url} className="card-img-top" alt="..." />
+        <div className="card-body text-center">
+          <h5 className="card-title">{name}</h5>
```
Comment (cwmanning) [inline src/pages/shop/Supplement.tsx:19]: this `h5` might be more appropriate as a `div`, especially if this is an "index" page and these are links within a list of many products

**Review by cwmanning — APPROVED** [review]
👍

**Review by nick-telsan — APPROVED** [review]

File: src/pages/shop/Supplement.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          <p className="card-text">${unit_price}</p>
```
Comment (solomonhawk) [inline src/pages/shop/Supplement.tsx:20]: In a larger-scale application, you might want to abstract this into a function called `formatPrice` or something like that which can centralize any logic required like rounding numbers or localization, etc.

File: src/pages/shop/shop.tsx [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
-  slug: string;
-  category: string;
-  description: string;
-  photo_url: string;
-  unit_price: number;
-  benefits: string[];
-};
-
 function Shop() {
   const query = useQuery(["supplements"], async () => await listSupplements());
```
Comment (solomonhawk) [inline src/pages/shop/shop.tsx:11]: Loading and error states?

**Review by solomonhawk — APPROVED** [review]

**Pull request merged**